Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine:3.18@sha256:82d1e9d7ed48a7523bdebc18cf6290bdb97b82302a8a9c27d4fe885949ea94d1
+
+ARG NAME=sbom-convert
+ENV NAME=${NAME}
+
+RUN apk add --no-cache \
+	bash \
+	docker-cli \
+	tini
+
+COPY scripts/entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
```

Comment:
I also prefer to he a static binary image way.
It also helps with easier to create multi arch images..

I would say about ko. 
I prefer the default docker go releaser maybe minor but annoying issues using goreleaser with ko, it won't build the image unless you actually release (so if we run a snapshot only the binary is built) further more it will force build to have actual access to registry ..

@purco can you elaborate on the chaingaurd base image it looks interesting ..

I do like us using sboms on our own tool, we should discuss this I think.. what and how..

Fyi, Goreleaser it self  also can connect to a sbom topl directly to run the sboms on the artifacts.

Reply:
Let's start with cleaning the dockerfile and then we can change to build with `ko`, I'll try changing the base alpine to cgr.dev/chainguard/static:sha***